torch/fx/experimental/symbolic_shapes.py

```diff
@@ -207,10 +207,10 @@ def __repr__(self):
 
     # These methods are metaprogrammed in below
     def sym_int(self) -> "SymNode":
-        ...
+        raise AssertionError("should have been overridden")
 
     def sym_float(self) -> "SymNode":
-        ...
+        raise AssertionError("should have been overridden")
```

Comment:
nit: Feel like a notimplemented?

Reply:
no, it's not that it's not implemented, this is just a stub that will get overwritten by metaprogramming later. If you ever actually reach this code that means something wrong happened in our code, therefore assert.